CCS/3. Tutorials/4. 確保計算工作能持續不中斷.md

```diff
@@ -1 +1,154 @@
+---
+
+{%hackmd @docsharedstyle/default %}
+
+# HowTo：Continuous and uninterrupted computing service
+
+
+- When we operate containers using SSH, the computing service may be disconnected due to the Internet. We provide the following 3 solutions to ensure that the computing work can continuously run in the background.
+
+
+## Method 1. Using Jupyter Notebook
+
+### Step 1. Click into the created Interactive Container in TWCC
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_248954d46886713138afa6f68486fa40.png)
+
+
+
+### Step 2. Launch the container using Jupyter Notebook 
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_c4b11ac00cb6e7dd5f7386b18c5a57ba.png)
+
+
+
+### Step 3. Click into Terminal to operate the container
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_a027717a19eb85582f5f893ec2ed999c.png)
+
+
+
+### Step 4. Insert and run the command
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_92e9382801d6dd746faeffa8043ddb86.png)
+
+
+:::info
+:bulb: The URL bar will display the terminal number: `terminals/1`, which will be used for the following steps.
+:::
+### Step 5. Check the status of the running program
+- If you close the Terminal, the program will run continuously in the background.
+- To check the running status of the program, click "Running" in Jupyter Notebook, and click the terminal number you want to check below.
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_b4acc84b64bb17c41533b4718a74bcc2.png)
+
+
+- Then you are able to enter Terminal again to check the status of the running program
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_62ab66f669dfb35b75aaec84c0ee1993.png)
+
+
+
+
+## Method 2. Using Screen 
+
+### Step 1. Install Screen
+
+- Launch the container using SSH, and then insert the command to install Screen:
+
+```bash=
+sudo apt-get install screen
+```
+
+:::info
+==**Need to update the link:exclamation:**==
+:book: Please refer to [<ins> Launch the container using SSH</ins>](https://www.twcc.ai/doc?page=container#使用-SSH-登入連線).
+:::
+:::info
+:warning: Warning: If the installation process shows `E: Unable to locate package screen`, please run the following command to update, then install Screen again.
+```bash=
+sudo apt-get update
+```
+:::
+
+
+### Step 2. Enter Screen 
+```bash=
+screen
+```
+- After entering Screen, please read the description.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_47056d9911ee362ef05bc9e6ac33febc.png)
+
+
+- Press "Return" to insert the command
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_e1b9bebfacbf19334f64d7f9c5b4cd7b.png)
+
+
+
+### Step 3. Run the program in the screen shell
+
+- The example is as follows
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_0017c6102df15b490d4492adf889470a.png)
+
+
+
+### Step 4. Detach screen shell
+
+- When the program is running, press `ctrl + a + d`, it will show detach successful message (shell ID marked in red frame). Then, the program can run continously in the background, even if SSH is disconnected.
```

Comment:
[SUG]

1. program*2 > processing
2. marked in red frame > marked with red frame